src/content/en/fundamentals/architecture/howto-components/howto-radio-group.md

```diff
@@ -0,0 +1,488 @@
+project_path: /web/_project.yaml
+book_path: /web/fundamentals/_book.yaml
+
+{# wf_updated_on: 2017-04-26#}
+{# wf_published_on: 2017-04-06 #}
+
+# HowTo: Components – howto-radio-group {: .page-title }
+
+{% include "web/_shared/contributors/ewagasperowicz.html" %}
+{% include "web/_shared/contributors/noms.html" %}
+{% include "web/_shared/contributors/robdodson.html" %}
+{% include "web/_shared/contributors/surma.html" %}
+
+<link rel="stylesheet" href="prism-solarizedlight.css">
+<link rel="stylesheet" href="main.css">
+
+<p>A <code>HowtoRadioGroup</code> is a set of checkable buttons, where only one button may
+be checked at a time. The <code>HowtoRadioGroup</code> element wraps a set of
+<code>HowtoRadioButton</code> children and manages their checked states in response to
+user keyboard actions such as pressing arrow keys to select the next radio
+button, or if the user clicks with a mouse.</p>
```

Comment:
This sentence needs some rework. Try: 'in response to user actions such pressing arrow keys, which selects an adjacent radio button, or clicking another radio button with a mouse.'

In any case, the verbs need to matching. Pressing-clicking instead of pressing-clicks.